tests/api/utils/data_classes.py

```diff
@@ -159,34 +163,60 @@ class ConfigLogFields(BaseClass):
 )
 
 
-class HostGroupFields(BaseClass):
+class GroupConfigHostCandidatesFields(BaseClass):
     """
-    Data type class for Host Group object
-    https://spec.adsw.io/adcm_core/objects.html#group
+    Data type class for GroupConfigHostCandidates object
     """
 
-    id = Field(name="id", f_type=PositiveInt(), default_value="auto")
-    host = Field(
-        name="host",
-        f_type=ForeignKey(fk_link=HostFields),
-        required=True,
-        postable=True,
-        changeable=True,
+    predefined_dependencies = [GroupConfigFields]
+
+    id = Field(
+        name="id",
+        f_type=PositiveInt(),
+        default_value="auto",
     )
-    group = Field(
-        name="group",
-        f_type=ForeignKey(fk_link=ConfigGroupFields),
+    cluster_id = Field(name="cluster_id", f_type=PositiveInt(), default_value="auto")
+    prototype_id = Field(name="prototype_id", f_type=PositiveInt(), default_value="auto")
+    provider_id = Field(name="provider_id", f_type=PositiveInt(), default_value="auto")
+    fqdn = Field(name="fqdn", f_type=String(), default_value="auto")
+    description = Field(name="description", f_type=String(), default_value="auto")
+    state = Field(name="state", f_type=String(), default_value="auto")
+    url = Field(name="url", f_type=String(), default_value="auto")
+
+
+GroupConfigFields.host_candidate = Field(
+    name="host_candidate",
+    f_type=BackReferenceFK(fk_link=GroupConfigHostCandidatesFields),
+    default_value="auto",
+)
```

Comment:
I suggest to leave a comment inside `GroupConfigFields` class declaration about this field. Or explicitly define `host_candidate` with special value as a default value (e.g. `defered` or make it obvious that field will be updated soon).

Reply:
done